dd-sdk-reactnative.podspec

```diff
@@ -17,5 +17,5 @@ Pod::Spec.new do |s|
   s.source_files = "ios/**/*.{h,m,mm,swift}"
   
   s.dependency "React-Core"
-  s.dependency "DatadogSDKBridge", "~> 0.4.3"
+  s.dependency "DatadogSDKBridge", "~> 0.4.4"
```

Comment:
in fact `~> 0.4.3` means `0.4.4` as of now.
do we still need this PR and a new `dd-sdk-reactnative` release?

Reply:
Ah, didn't know. I would say yes, because this PR updates podfile for example app anyway, which is used by us to test features.